src/chrome/content/rules/Copenhagenmarathon.dk.xml

```diff
@@ -1,12 +1,8 @@
-<!--
-	Mixed content from $self (secured in Copenhagenmarathon.dk-falsemixed.xml):
-		map.copenhagenmaraton.dk
-
--->
 <ruleset name="Copenhagenmarathon.dk">
 
 	<target host="copenhagenmarathon.dk" />
 	<target host="www.copenhagenmarathon.dk" />
+	<target host="map.copenhagenmarathon.dk" />
```

Comment:
Could you also add `2018.copenhagenmarathon.dk` and `resultater.copenhagenmarathon.dk`?

Reply:
Sure. I've added 2018 before map but after www. I hope I'm interpreting the sorting rules correctly.